app/model/SectionAudit.scala

```diff
@@ -20,6 +21,16 @@ case class SectionAudit(
                      sectionSummary: SectionSummary
                    ) {
   def toItem = Item.fromJSON(Json.toJson(this).toString())
+
+  def asAuditingThrift = Notification(
+    app = App.TagManager,
+    operation = operation,
+    userEmail = s"${user.replace(" ", ".").toLowerCase}@guardian.co.uk",
+    date = date.toString,
+    resourceId = Some(sectionId.toString),
+    message = Some(s"${description}. Section: ${sectionSummary.toString}"),
+    shortMessage = Some(s"${description} Section ID: ${sectionId.toString}")
```

Comment:
it's up to you to decide what to do with `message` and `shortMessage`, but maybe you only need `Some(s"${description}")`, the `sectionId` is already inside `resourceId`